### ISSUE TITLE
The payload is not included in the ActiveNotifications

### ISSUE THREAD

**Issue by bugnano** [MaikuB/flutter_local_notifications#1596]
**Describe the bug**

The `FlutterLocalNotificationsPlugin.show()` method has a `payload` parameter, which, at the moment, is useful only when the user interacts with the notification.

By contrast, I need more information than just `title` and `body` of the already active notifications when I start the app. So I was thinking about using the `payload` of the notification to get that information.

The problem is that the `ActiveNotification` class has no extra fields of information other than `title` and `body`. It would be really useful to have access to the `payload` information as well (or some other kind of data that I can put in the notification, and then retreive at application startup, even if it's only for Android 8 or newer).

**Comment by MaikuB** [comment]
This is available in the 10.0.0 pre-release

**Comment by MichalNemec** [comment]
@MaikuB Hello, i think its no longer working (^19.3.0):
```
await flutterLocalNotificationsPlugin.show(
      data,
      data.title,
      data.body,
      data.notificationDetails,
      payload: jsonEncode(message.data),
    )
Future<List<ActiveNotification>> getActiveNotifications() async {
    if (kIsWasm || kIsWeb) return [];
    return await flutterLocalNotificationsPlugin.getActiveNotifications();
  }
```
When i put breakpoint on the return of active notifications, i get this, where its really clear that payload is not set.

<img width="323" height="223" alt="Image" src="https://github.com/user-attachments/assets/bc8ef713-ed0d-4885-8499-0a4b4adcad77" />
